pkg/agent/util/net_windows.go

```diff
@@ -466,3 +466,12 @@ func HostInterfaceExists(ifaceName string) bool {
 	}
 	return true
 }
+
+// SetInterfaceMTU configures interface MTU on host for Pods. MTU change cannot be realized with HNSEndpoint and API because
+// 1. There's no MTU field in HNSEndpoint: https://github.com/Microsoft/hcsshim/blob/4a468a6f7ae547974bc32911395c51fb1862b7df/internal/hns/hnsendpoint.go#L12
+// 2. Container MTU configuration does not work on Windows: https://github.com/moby/moby/issues/35683
```

Comment:
what does this mean? Is there a container MTU configuration somewhere?

Reply:
+1 for adding some clarification here.

Reply:
I had a second thought and I misunderstood something. This line is removed.